docs/apis-tools/console-api/tutorial.md

```diff
@@ -0,0 +1,263 @@
+
+```
+console.log(
+      `Client added! Name: ${newClient.name}. ID: ${newClient.clientId}.`
+    );
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+```
+
+6. In your terminal, run `npm run cli admin add` to create your new client.
+
+## GET a client ID
+
+To get a client ID, take the following steps:
+
+1. Outline your function, similar to the steps above:
+
+```
+async function viewClient([clientId])
+  const accessToken = await getAccessToken();
+
+  const administrationApiUrl = process.env.ADMINISTRATION_API_URL;
+  const clusterId = process.env.CLUSTER_ID;
+```
+
+2. Write the API endpoint to view a single client within a cluster:
+
+```
+const url = `${administrationApiUrl}/clusters/${clusterId}/clients/${clientId}`;
+```
+
+3. Call the client endpoint using a GET method:
+
+```
+var options = {
+    method: "GET",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+```
+
+4. Process your results from the API call and emit the client details:
+
+```
+try {
+    const response = await axios(options);
+
+    const clientResponse = response.data;
+
+    console.log("Client:", clientResponse);
+  } catch (error) {
+    console.error(error.message);
+  }
+```
+
+5. In your terminal, run `npm run cli admin view` to view your client.
+
+## DELETE a client
+
+To delete a client, take the following steps:
+
+1. Outline your function, similar to the steps above:
+
+```
+async function deleteClient([clientId]) {
+  const accessToken = await getAccessToken();
+
+  const administrationApiUrl = process.env.ADMINISTRATION_API_URL;
+  const clusterId = process.env.CLUSTER_ID;
+
+  const url = `${administrationApiUrl}/clusters/${clusterId}/clients/${clientId}`;
+```
+
+2. Configure the API call using the DELETE method:
+
+```
+  var options = {
+    method: "DELETE",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+```
+
+3. Process the results from the API call. For example:
+
+```
+try {
+    const response = await axios(options);
+
+    if (response.status === 204) {
+      console.log(`Client ${clientId} was deleted!`);
+    } else {
+      console.error("Unable to delete client!");
+    }
+  } catch (error) {
+    console.error(error.message);
+  }
+```
+
+4. In your terminal, run `npm run cli admin delete` to delete your client.
+
+## Next steps
+
+Having trouble configuring your API calls or want to examine an example of the completed tutorial? Navigate to the `completed` folder in the [GitHub repository](https://github.com/camunda/camunda-api-tutorials/tree/main/completed), where you can view an example `administration.js` file.
+
+As mentioned, you can also script several additional API calls as outlined in the [Administration API reference material](/apis-tools/administration-api/administration-api-reference.md).
```

Comment:
```suggestion
## If you get stuck

Having trouble configuring your API calls or want to examine an example of the completed tutorial? Navigate to the `completed` folder in the [GitHub repository](https://github.com/camunda/camunda-api-tutorials/tree/main/completed), where you can view an example `administration.js` file.

## Next steps

You can script several additional API calls as outlined in the [Administration API reference material](/apis-tools/administration-api/administration-api-reference.md).
```

What do you think about separating these paragraphs into different sections? The `completed` folder is more of a hint for when they get stuck _on this tutorial_, not something they would try next.